scripts/release_info.py

```diff
@@ -139,7 +143,7 @@ def extract_pr_numbers(logs):
         set(int): Set of PR numbers extracted from the log.
     """
     pr_numbers = PR_NUMBER_REGEX.findall(
-        ' '.join([log.message for log in logs]))
+        ' '.join([log.message.split('\n')[0] for log in logs]))
```

Comment:
Can we just do one more thing to the regex? Can we make sure that the (#XXXX) is at the end of the line? (Add a $ at the end of the regex). This would make the regex not match (#xxxx) patterns in the title of the PR.

Reply:
Done.